Assets/MRTK/Providers/OpenXR/MRTK.OpenXR.asmdef

```diff
@@ -30,8 +30,13 @@
         },
         {
             "name": "com.microsoft.mixedreality.openxr",
-            "expression": "[0.1.0,0.1.3)",
-            "define": "MSFT_OPENXR_PRE_013"
+            "expression": "0.2.0",
```

Comment:
Does this expression work for everything greater than 0.2.0?

Reply:
It does! This is valid for any version >= the expression:

![image](https://user-images.githubusercontent.com/3580640/112540876-eeaaf680-8d6f-11eb-9b68-437b757a61e4.png)